pkg/apis/imageregistry/v1/types.go

```diff
@@ -11,6 +11,9 @@ import (
 
 const (
 
+	// DefaultRouteName is the name of the default route created for the registry
+	// when a default route is requested from the operator
+	DefaultRouteName = "default-route"
```

Comment:
Isn't this just a prefix, not the whole route name?

Reply:
it is the route name.  it is not the route hostname.

Reply:
("oc get route default-route")